lib/chef/provider/package/cab.rb

```diff
@@ -45,7 +45,7 @@ def remove_package(name, version)
         end
 
         def dism_command(command)
-          shellout = Mixlib::ShellOut.new("dism.exe /Online #{command} /NoRestart", { :timeout => @new_resource.timeout })
+          shellout = Mixlib::ShellOut.new("dism.exe /Online /English #{command} /NoRestart", { :timeout => @new_resource.timeout })
```

Comment:
Sorry, i don't get the point, why we can't use this switch by default like chocolatey does
https://github.com/chocolatey/choco/blob/078cd3219c75328802c862cde9ee5e884e0478ca/src/chocolatey/infrastructure.app/services/WindowsFeatureService.cs#L139